src/components/createElementComponent.test.tsx

```diff
@@ -165,6 +167,12 @@ describe('createElementComponent', () => {
       false
     );
 
+    const PaymentFormElement = createElementComponent(
+      // @ts-expect-error - `paymentForm` Element type is not part of stripeJs.StripeElementType yet
```

Comment:
necessary until https://github.com/stripe/stripe-js/pull/829 is merged and deployed